v2/passwordless/updating-supertokens.mdx

```diff
@@ -0,0 +1,54 @@
+---
+id: updating-supertokens
+title: Updating Supertokens SDKs and Core
+hide_title: true
+---
+
+<!-- COPY DOCS -->
+<!-- ./community/updating-supertokens.mdx -->
+
+# Updating Supertokens SDKs and Core
+
+Not all older releases of every supertokens Core, front-end SDK, and back-end SDK are compatible with each other. In order to maintain compatability
+and keep up on new features, we recommend updating and upgrading all of your SDKs regularly, starting with the supertokens core, then the back-end SDK, then the front-end SDK.
+
+The following is a link to each core and SDK changelog, containing steps to migrate from each version. You can use our [compatability table](../../community/compatibility-table) to see which versions you need to upgrade to.
```

Comment:
if you visit this link on any of the recipe guides, its broken..

Reply:
I had hardcoded the link to the updating-supertokens page before and forgot to update this after I set up page copying, good catch. I'll fix this